### ISSUE TITLE
Installation from OperatorHub.io fails

### ISSUE THREAD

**Issue by Avni-Sharma** [redhat-developer/service-binding-operator#960]
### Minikube version 
v1.16.0

### What is the SBO version used?
0.7.1

### What are the steps to reproduce this issue?
Follow the steps to install sbo on minikube from https://operatorhub.io/operator/service-binding-operator

### What is the expected behaviour?
CSV of the SBO should be present and operator pod running

### What is the actual behaviour?
Failing subscription
```
status:
  catalogHealth:
  - catalogSourceRef:
      apiVersion: operators.coreos.com/v1alpha1
      kind: CatalogSource
      name: operatorhubio-catalog
      namespace: olm
      resourceVersion: "911"
      uid: 20f17516-805c-4fd0-805a-fcab38752723
    healthy: true
    lastUpdated: "2021-05-04T11:29:23Z"
  conditions:
  - lastTransitionTime: "2021-05-04T11:29:23Z"
    message: all available catalogsources are healthy
    reason: AllCatalogSourcesHealthy
    status: "False"
    type: CatalogSourcesUnhealthy
  lastUpdated: "2021-05-04T11:29:23Z"
  ```

### Additional Information (Screenshots, etc)
```
k get subscriptions --all-namespaces
NAMESPACE   NAME                          PACKAGE                    SOURCE                  CHANNEL
operators   my-service-binding-operator   service-binding-operator   operatorhubio-catalog   beta
```

**Comment by pedjak** [comment]
@Avni-Sharma can you provide some more details? Like are pods created? I am not able to reproduce it on my instance.

**Comment by Avni-Sharma** [comment]
kubectl get packagemanifest | grep service-binding-operator
service-binding-operator                   Community Operators   6m4s

**Comment by Avni-Sharma** [comment]
For OLM I used the step mentioned on the operatorhub.io
i.e 
` curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.18.0/install.sh | bash -s _v0.18.0`

OLM - 0.18.0

**Comment by pedjak** [comment]
Oh, so the issue is with OLM 0.18, with 0.17 everything works fine. The problem is on OLM side, already reported in https://github.com/operator-framework/operator-lifecycle-manager/issues/2129

Until the issue is fixed, we should use 0.17.0
